networking/hardware_networks/switching-bf2-nic-dpu.adoc

```diff
@@ -0,0 +1,18 @@
+:_content-type: ASSEMBLY
+[id="switching-bf2-nic-dpu"]
+= Switching Bluefield-2 from DPU to NIC
+include::_attributes/common-attributes.adoc[]
+:context: switching-bf2-nic-dpu
+
+toc::[]
+
+You can switch the Bluefield-2 network device from data processing unit (DPU) mode to network interface controller (NIC) mode. 
+
+:FeatureName: Switching Bluefield-2 from data processing unit (DPU) mode to network interface controller (NIC) mode
+include::snippets/technology-preview.adoc[]
+
+include::modules/proc-switching-bf2-nic.adoc[leveloffset=+1]
+
+.Additional resources 
```

Comment:
Usually this needs the additional resources tag above it, but not super critical